Ppap.scala

```diff
@@ -0,0 +1,10 @@
+object Main extends App{
+  import scala.util.Random
+  
+  def ppap(pp: List[String] = Nil): Unit = pp match {
+    case "pen" :: "pineapple" :: "apple" :: "pen" :: _ => println(pp.reverse.mkString + " Ah-! Pen Pineapple Apple Pen")
```

Comment:
Although I don't know about PPAP program,  should it be 

```scala
case "Pen" :: "Pineapple" :: "Apple" :: "Pen" :: _ => println(pp.reverse.mkString + " Ah-! Pen Pineapple Apple Pen")
case _ => ppap((if(Random.nextBoolean) "Pen" else if(Random.nextBoolean) "Pineapple" else "Apple") :: pp)
```

?

It seems that the first letters are wrong.